modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/configuration/schema/PersistPageMemoryStorageEngineExtensionConfigurationSchema.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.storage.pagememory.configuration.schema;
+
+import org.apache.ignite.configuration.annotation.ConfigValue;
+import org.apache.ignite.configuration.annotation.ConfigurationExtension;
+import org.apache.ignite.internal.storage.configurations.StorageEngineConfigurationSchema;
+
+/**
+ * Storages configuration extension for persistent page memory storage engine.
+ */
+@ConfigurationExtension
+public class PersistPageMemoryStorageEngineExtensionConfigurationSchema extends StorageEngineConfigurationSchema {
```

Comment:
Why is this class even needed?

Reply:
This class needed to provide the concrete storage engine configs under the parent "storage" root. Eariler these configs was the root configs, but now we need to make it inner. At the same time we can't use NamedConfigValue+PolymorphicConfig, please look at the https://github.com/gridgain/apache-ignite-3/pull/92#discussion_r1474897624